### DISCUSSION TITLE
Where the configuration save on macOS

### DISCUSSION THREAD

**Discussion by tshu-w** [brookhong/Surfingkeys#1749] [Q&A] [answered]
I somehow lost my surfingkeys configuration. I want to know where the configs save on macOS. I try to use find, but cannot find it.

**Comment by brookhong** [comment, marked as answer]
It is not a standalone file that can be found on your disk, it's in `chrome.storage.local`, which is managed by your browser, and stored in `~/Application Support/Google/Chrome/Default/Local Storage/`.

The built-in short cut from Surfingkeys to export your configuration is `yj`, which will dump your configuration into system clipboard, then you can paste in some text editor. But it probably does not help, since you have lost your configurations.

**Reply by tshu-w** [reply]
@brookhong Sorry for not being clear, but is this the same for Safari?

**Reply by brookhong** [reply]
Almost, except the folder for Safari should be `~/Library/Safari/LocalStorage/`.

**Reply by tshu-w** [reply]
But I cannot find some file name contains surfingkeys at `~/Library/Safari/LocalStorage/`
```console
~
λ (cd ~/Library/Safari/LocalStorage/ && fd surf)

```
